identifiers/country-hr.csv

```diff
@@ -1,13 +1,14 @@
-ocd-division/country:hr,Hrvatska
-ocd-division/country:hr/ed:1,I. Izborna jedinica
-ocd-division/country:hr/ed:2,II. Izborna jedinica
-ocd-division/country:hr/ed:3,III. Izborna jedinica
-ocd-division/country:hr/ed:4,IV. Izborna jedinica
-ocd-division/country:hr/ed:5,V. Izborna jedinica
-ocd-division/country:hr/ed:6,VI. Izborna jedinica
-ocd-division/country:hr/ed:7,VII. Izborna jedinica
-ocd-division/country:hr/ed:8,VIII. Izborna jedinica
-ocd-division/country:hr/ed:9,IX. Izborna jedinica
-ocd-division/country:hr/ed:10,X. Izborna jedinica
-ocd-division/country:hr/ed:11,XI. Izborna jedinica - Dijaspora
-ocd-division/country:hr/ed:12,XII. Izborna jedinica - Nacionalne Manjine
+id,name
+ocd-division/country:hr,Hrvatska
+ocd-division/country:hr/ed:1,I. Izborna jedinica
+ocd-division/country:hr/ed:10,X. Izborna jedinica

```

Comment:
One little nit-pick, I see this file got sorted, but can we sort the electoral districts by number like they were before?

Reply:
It is generated directly by the script compile.py

Reply:
Oh i see, nevermind then!